SpectraLogic.EscapePodClient/Exceptions/MemberAlreadyPartOfClusterException.cs

```diff
@@ -21,16 +21,16 @@ namespace SpectraLogic.EscapePodClient.Exceptions
     ///
     /// </summary>
     /// <seealso cref="System.Exception" />
-    public class MemberAlreadyPartOfClusterException : Exception
+    public class AlreadyPartOfClusterException : Exception
```

Comment:
To be consistent I would say something to the effect of, `AlreadyAClusterMemberException`